stan/math/prim/arr/fun/value_of.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef STAN_MATH_PRIM_ARR_FUN_VALUE_OF_HPP
+#define STAN_MATH_PRIM_ARR_FUN_VALUE_OF_HPP
+
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/meta/child_type.hpp>
+#include <vector>
+#include <cstddef>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * Convert a std::vector of type T to a std::vector of doubles.
```

Comment:
fix comment to reflect that it'll return a vector of `child_type<T>::type`, not double